spring-cloud-azure-context/src/main/java/com/microsoft/azure/spring/cloud/context/core/impl/RedisCacheManager.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.cloud.context.core.impl;
+
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.redis.RedisCache;
+import com.microsoft.azure.spring.cloud.context.core.config.AzureProperties;
+
+public class RedisCacheManager extends AzureManager<RedisCache, String> {
+
+    public RedisCacheManager(Azure azure, AzureProperties azureProperties) {
+        super(azure, azureProperties);
+    }
+
+    @Override
+    String getResourceName(String key) {
+        return key;
+    }
+
+    @Override
+    String getResourceType() {
+        return RedisCache.class.getSimpleName();
```

Comment:
I am not sure if we can define enum for ResourceType instead of class simpleNames here.

Reply:
For simplicity now, no need to define another enum.